cli/tests/functional/src/test/resources/io/helidon/tests/functional/input.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
What's the use of this empty file ? Is that simulating three newlines ?

Reply:
Yes, I did the same than test from impl module. Nevertheless, I removed this file and the tests are creating a temporary file instead.

Reply:
Yes, I did the same than the `impl` module test. Nevertheless, I removed this file and the test create a temporary file instead.